src/pages/BrowseSchemes.svelte

```diff
@@ -164,6 +165,7 @@
           <CollapsibleCard label="Road network">
```

Comment:
Not sure if this category name is still the best, or how we'll group things as we pull in more

Reply:
Agreed it seemed a little odd to me but nowhere better for now. I guess in future we might want a public transport card

Reply:
Train stations could be both in public transport and trip generators